pho/src/components/Pomodoro/Pomodoro.tsx

```diff
@@ -1,12 +1,23 @@
 import { Box, Center } from '@chakra-ui/react';
-import React from 'react';
+import React, { useState } from 'react';
 import Draggable from 'react-draggable';
 
 import { usePomodoro } from '@/hooks/usePomodoro';
 
 import sessionIcon from '../Pomodoro/session.png';
 import './Pomodoro.css';
 
+const getWindowDimensions = () => {
+  const { innerWidth: width, innerHeight: height } = window;
+  return {
+    width,
+    height,
+  };
+};
+
+const pomodoroWidthPixels = 250;
+const toolbarRightPosition = 62;
```

Comment:
Please put this function inside `lib` folder as. I think it should be considered as utility function. @thienanhvu

Reply:
Resolved! Thank you